templates/galaxy/config/tool_conf.xml.j2

```diff
@@ -643,4 +634,16 @@
     <tool file="interactive/interactivetool_mgnify_notebook.xml" />
     <tool file="interactive/interactivetool_mgnify_notebook_1.2.2.xml" />
   </section>
+
+<!-- Ecology subdomain -->
+<label id="ecology-data-management" text="Data management" />
```

Comment:
Is this one new?
```suggestion
<label id="ecology-data-management" text="Ecology Data Management" />
```

Reply:
We already have "data_and_metadata_management" isn't it?

Reply:
Yes, don't know why I didn't find it...